### DISCUSSION TITLE
Ignore errors on TM1 Chore Execution

### DISCUSSION THREAD

**Discussion by NuwanGebalanage** [cubewise-code/tm1py#769] [Q&A] [answered]
Hi,

We have recently started using TM1py for our chore automation where as  they were running on a schedule in the past. 
One thing we've noticed is that when there is an error, the TM1 Chore on schedule works fine but the TM1py chore execution fails. 
Any pointers on ignore error when executing TM1 chores using TM1 Py?

**Comment by MariusWirtz** [comment, marked as answer]
Hi @NuwanGebalanage,

I can not reproduce your finding.

I have a chore with 3 processes of which one process raises minor errors and two processes succeed.

``` python
from TM1py.Services.TM1Service import TM1Service

with TM1Service(base_url="https://localhost:12354", user="admin", password="apple") as tm1:
    response = tm1.chores.execute_chore("chore_with_process_errors")
```

When I use TM1py to execute the chore, all 3 processes execute. See the code and logs below

<img width="938" alt="image" src="https://user-images.githubusercontent.com/15182131/180186852-b1b25962-ed85-4a5f-b9c8-5b38cd4a13fb.png">

**Reply by NuwanGebalanage** [reply]
Thanks for the quick response Marius. Will try to see if I can find the logs

**Reply by MariusWirtz** [reply]
Hi @NuwanGebalanage,

has the issue been resolved?
